config/api/src/main/java/org/jboss/windup/config/metadata/MetadataBuilder.java

```diff
@@ -141,7 +141,9 @@ public static MetadataBuilder forProvider(Class<? extends RuleProvider> implemen
             {
                 builder.addTargetTechnology(new TechnologyReference(
                             technology.id(),
-                            Versions.parseVersionRange(technology.versionRange())));
```

Comment:
What is this about?

Reply:
I ported the fix for handling the [`versionRange` default `""`](https://github.com/windup/windup/blob/4.0.x/config/api/src/main/java/org/jboss/windup/config/metadata/Technology.java#L25) implemented some lines above for [source technologies](https://github.com/windup/windup/blob/4.0.x/config/api/src/main/java/org/jboss/windup/config/metadata/MetadataBuilder.java#L131) to the target technologies.

Reply:
Ah, that makes sense.